### PR TITLE
Improve sharing tab style & add icons

### LINKED ISSUE #6570: Design: Shares overview Tabs
Please refine the tabs style in shares: 

# Current style:
image

# Desired style:
image

image

### PR DESCRIPTION
**Pull request by pascalwengerter**
## Related Issue
- Fixes #6570

### Desktop/Tablet
Screenshot 2022-03-11 at 12-02-47 Files shared with me - ownCloud

### Mobile
Screenshot 2022-03-11 at 12 03 10

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>oc10SharingIntUsers3</strong> https://drone.owncloud.com/owncloud/web/23522/31/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUISharingInternalUsersShareWithPage-shareWithUsers_feature-L59.png
  


</p></details>
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by kulmann** [comment]
XPath selector `//nav[@id="shares-navigation"]/ul/li/a[.="Shared with others"` doesn't work anymore because it also includes an icon now. Need to go down to the span. But the selector looks a bit too strict to me, can be less strict like this:
`//nav[@id="shares-navigation"]//*[.="Shared with others"]`

**Comment by kulmann** [comment]
Could you include a screenshot in the PR description?

**Comment by pascalwengerter** [comment]
> Could you include a screenshot in the PR description?

Done

**Comment by pascalwengerter** [comment]
> XPath selector `//nav[@id="shares-navigation"]/ul/li/a[.="Shared with others"` doesn't work anymore because it also includes an icon now. Need to go down to the span. But the selector looks a bit too strict to me, can be less strict like this: `//nav[@id="shares-navigation"]//*[.="Shared with others"]`

Included, good hint, thx!

File: packages/web-app-files/src/components/AppBar/SharesNavigation.vue [hunk 1]
```diff
@@ -56,17 +61,21 @@ export default {
       return routes
     }, {})
     const navItems = computed(() => [
+      // computed aka not reactive?
```
Comment (kulmann) [inline packages/web-app-files/src/components/AppBar/SharesNavigation.vue:64]: dev leftover? computed -> immutable (at least not via setting a value)

Comment (pascalwengerter) [reply packages/web-app-files/src/components/AppBar/SharesNavigation.vue:64]: Meh, that was my reminder to investigate what later "magically" worked 🤷🏽 removing it

**Review by kulmann — APPROVED** [review]
LGTM 👍

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingInternal2</strong> https://drone.owncloud.com/owncloud/web/23586/60/1
The following scenarios passed on retry:
- webUISharingInternalUsers/shareWithUsers.feature:342

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingPublic2</strong> https://drone.owncloud.com/owncloud/web/23586/68/1
The following scenarios passed on retry:
- webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature:93

**Comment by ownclouders** [comment]
Results for <strong>oC10IntegrationApp1</strong> https://drone.owncloud.com/owncloud/web/23586/71/1
The following scenarios passed on retry:
- webUIPrivateLinks/accessingPrivateLinks.feature:9

**Pull request merged**